docs/concepts/jcloud/yaml-spec.md

```diff
@@ -34,32 +32,46 @@ Since each Executor has its own business logic, it may require different cloud r
 In JCloud, you can pass highly customizable, finely-grained resource requests for each Executor using the `jcloud.resources` argument in your Flow YAML.
 
 
-### CPU
+### Instance
 
-By default, `0.1 (1/10 of a core)` CPU is allocated to each Executor. You can use the `cpu` argument under `resources` to change it.
+JCloud uses the concept of "Instance" to respresent the hardware specifications. 
+In above example, C2 Instance Type stands for 4 cores and 8G RAM based on the CPU Tiers Instance defination table below.
```

Comment:
```suggestion
In above example, a C2 Instance Type represents four cores and 8 GB RAM based on the CPU Tiers Instance definition table below.
```

Please:

- Spell check throughout
- Switch `8G` to `8 GB` (with space) throughout

Reply:
cool fixed